src/renderer/gl_engine/tvgGlTessellator.cpp

```diff
@@ -1778,6 +1789,18 @@ void Stroker::strokeLineTo(const Point& curr)
 
 void Stroker::strokeCubicTo(const Point& cnt1, const Point& cnt2, const Point& end)
 {
+    if (mStrokeState.prevPt == end) {
```

Comment:
```
    // special case for line with zero length
    if (mStrokeState.prevPt == curr) {
        if (mStrokeCap == StrokeCap::Butt && tvg::zero(mStrokeWidth)) return;
        if (mStrokeCap == StrokeCap::Square) strokeSquarePoint(curr);
        else if (mStrokeCap == StrokeCap::Round) strokeRoundPoint(curr);
    }
```

Reply:
Here, we don't have curr. However, we likely need to compare prevPt and end instead.